mirrord/agent/src/file.rs

```diff
@@ -412,7 +412,7 @@ impl FileManager {
     }
 
     /// Handles our `readlink_detour` with [`std::fs::read_link`].
-    #[tracing::instrument(level = "trace", skip_all)]
+    #[tracing::instrument(level = Level::TRACE, skip_all, err(level = Level::ERROR))]
```

Comment:
```suggestion
    #[tracing::instrument(level = Level::TRACE, skip_all)]
```
This should not be logged globally as an agent error, user might simply use a non-existent path